views/help.php

```diff
@@ -0,0 +1,591 @@
+                    <h5 id="classes-archives">Restore / Archived Classes</h5>
+                    <div>
+                        <p>
+                            When a class is "deleted" it is actually sent to the archive where it can be either
+                            restored or fully deleted from the system. To access the <a href="/classes/archive">archive</a>,
+                            go to the main <a href="/classes">classes page</a>, and then click the "See Archive" button
+                            in the top left corner of the page.
+                        </p>
+                        <p>
+                            <i><span class="text-muted">Restore</span></i><br />
+                            To restore a class, click the "Restore" button beneath the desired class.
+                            <i>Note: All curricula previously associated with the class will still be linked.</i>
+                        </p>
+                        <p>
+                            <i><span class="text-muted">Full Delete</span></i><br />
+                            To fully delete a class, click the "Delete" button beneath the desired class.
+                            <b>If there is attendance based on this class, this information will be removed.</b>
+                        </p>
+                    </div>
+                <?php } ?>
+            <?php } ?>
+
+            <?php if (hasRole(Role::Coordinator)) { ?>
+                <h4 id="reports">Reports</h4>
+                <h5 id="reports-monthly">Monthly</h5>
+                <div>
+                    <p>
+                        The Monthly Reports page provides the demographics of participants served.  It also displays
+                        the amounts of participants served based on their home zip code.
+                    </p>
+                    <p>
+                        Click the left drop down, to select the month that you would like to generate a report for and
+                        the right drop down to select the year. Once the appropriate selections have been made, click
+                        the "Generate Report" button.  The Monthly Report results will be displayed on the same page.
+                        It displays:
+                    </p>
+                    <ul>
+                        <li>Month and year being reported on</li>
+                        <li>Amount of participants served</li>
+                        <li>Participant ethnicity</li>
+                        <li>Amount of participants served organized by their home zip code</li>
+                    </ul>
+                </div>
+                <h5 id="reports-quarterly">Quarterly</h5>
+                <div>
+                    <p>
+                        The Quarterly Reports page provides you with participant totals and survey results for the
+                        Quarter chosen. Participant totals show the amount to unduplicated Adults and Children served
```

Comment:
`totals show the amount to unduplicated` -> `totals show the amount of unduplicated`